generators/component/templates/custom/style.styl

```diff
@@ -1,7 +1,8 @@
-$rupture.scale = lookup('$global-layout-scale') || (0 400px 800px 1200px)
-$rupture.anti-overlap = lookup('$global-layout-overlap') || -1px
-$gutterWidth = lookup('$global-layout-gutterWidth') || 15px
-$containerMaxWidth = lookup('$global-layout-containerMaxWidth') || 1140px
+$rupture.scale = lookup('$globalLayoutScale') || (0 400px 800px 1200px)
```

Comment:
please also update the default scale to the latest standard (should use all possible rupture breakpoints)